src/helm/blue-agent/templates/_helpers.tpl

```diff
@@ -166,6 +166,9 @@ containers:
               rm -rf ${TMPDIR};
               {{- end }}
             {{- end }}
+            {{- range .Values.pipelines.customPreStopCommand }}
+            - "{{ . }}"
```

Comment:
If the abode conditions are `false` and the new variable is kept as its default, the same bug will occur. This will force users to debug the install process for a dumb issue.

Plus, note that the issue should appear on Windows.

I propose adding final `else` statements for each block (Windows and Linux) with something like `true` instead of the command. This is simpler and will work out of the box.